website/docs/r/resource_management_private_link.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+subcategory: "Management"
```

Comment:
this should be within `Resources` (since it's tied to a Resource Group)

Reply:
![image](https://github.com/hashicorp/terraform-provider-azurerm/assets/104055472/6e404fbb-61fa-4321-87b7-a6e1a6d748c5)

From [architecture doc](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/create-private-link-access-commands?tabs=azure-cli#understand-architecture), it is currently only supported at Management Group level, so could we put under `Management` subcategory? 
On the other hand, there is no `Resources` category: https://github.com/hashicorp/terraform-provider-azurerm/blob/b4bd9860cffeee2231e2bb51ba1a0e2b80f083f9/internal/services/resource/registration.go#L24-L31